Flesh Out Offline First Strategy Content

bradley-holt opened the pull request:
This pull request:

* Fleshes out the Offline First strategy page which provides:
  * A high-level conceptual overview
  * Some of the key benefits
  * The connection to Mobile First
  * The connection to IBM Cloudant and IBM Watson Data Platform
  * A reference to the broader Offline First community outside of IBM
  * A reference to reproducible data science and cognitive and analytics at the edge 
* Adds an Offline First apps feature page which:
  * Dives deeper into building Offline First apps
  * Features Progressive Web Apps but also covers hybrid mobile apps, native mobile apps, desktop apps, and Internet of Things (IoT) apps
  * Allows the Offline First strategy page to stay focused on providing a high-level conceptual overview
* Fleshes out the Location Tracker feature page
* Adds a Cloudant FoodTracker feature page
* Adds Voice of InterConnect Medium post to the list of projects
* Adds Offline Sync for Progressive Web Apps Medium post to the list of projects
* Adds Cloudant FoodTracker GitHub repo to the list of projects
* Adds a signal bars graphic for use on the the Offline First strategy page
* Adds a `CREDITS.md` for tracking image sources
* Adds a Offline First apps graphic for use on the Offline First apps feature page

Notes:

* The default sort order prioritizes Cloudant FoodTracker and Location Tracker above Offline Camp and Offline First App Development on the homepage and on the Offline First feature page. The ideal sort order here would be Offline Camp, Offline First App Development, Location Tracker, and then Cloudant FoodTracker last.

bradley-holt committed:
Add signal bars graphic

bradley-holt committed:
Add CREDITS.md file for capturing image credits, etc.

bradley-holt committed:
Add Offline First apps graphic

bradley-holt committed:
Flesh out content for Offline First strategy page

bradley-holt committed:
Add Offline First apps feature page

bradley-holt committed:
Add Voice of InterConnect project

bradley-holt committed:
Add Cloudant FoodTracker as a feature

bradley-holt committed:
Update Offline First strategy page wording and add hyperlinks

bradley-holt committed:
Update Offline First apps feature page wording and add link to Cloudant FoodTracker

bradley-holt committed:
Flesh out Location Tracker feature page

bradley-holt committed:
Add Offline Sync for Progressive Web Apps to projects

bradley-holt committed:
Flesh out Cloudant FoodTracker feature page

bradley-holt committed:
Add Cloudant FoodTracker GitHub repo to projects

bradley-holt committed:
Set Offline First apps feature page as featured

terichadbourne commented:
Just a heads up that with @bradley-holt's headline and the CSS that was present at the time Bradley started this branch, the headline fills up (or fills up more than the entire first screen you see on both a Mac window and a phone: 

full mac window:
![image](https://user-images.githubusercontent.com/19171465/29423326-4bf4ac30-8349-11e7-83bd-6581c6bb5e3e.png)

phone:
![image](https://user-images.githubusercontent.com/19171465/29423164-b8af5d9e-8348-11e7-9387-f2271f79417c.png)

The master (which has a previous and shorter headline) looks better, but I'm not positive whether it's because @vabarbosa  has already made site-wide CSS updates to shrink headlines or if it's about the length of text here.  Here's what I see when in the master, which has old content but perhaps current styling? 

full mac window:
![image](https://user-images.githubusercontent.com/19171465/29423378-889d3940-8349-11e7-9fb5-9e912a0b6c9c.png)

phone: 
![image](https://user-images.githubusercontent.com/19171465/29423431-b778db84-8349-11e7-9473-8a640cbb3e35.png)

If the styling has changed since Bradley started his branch, this may work itself out when the branch is merged. If not, the website might be built with shorter headlines in mind. 

cc @mikebroberg @bradnoble

vabarbosa commented:
@terichadbourne there hasnt been any recent changes to headline size. however, the site's font has changed from Google's Roboto to IBM's Plex, but there isnt really a size discrepancy between the two fonts. i think it is the shorter headline, which always helps and should definitely be taken into consideration.

mikebroberg committed:
Edit offline first strategy copy

I cut it way down, but tried to keep with your initial points. I also
tried to not define offline first by saying what it _isn’t_, which
seemed to be a problem with the temporary text we had up there.

mikebroberg committed:
More edits to offline first strategy

I cut the Mobile First stuff. I like it, but it just felt too long to
get to the collections content.

mikebroberg committed:
Fix typo in headline

mikebroberg committed:
Fix typo in FoodTracker feature

mikebroberg committed:
Merge branch 'master' into offline-first-strategy

# Conflicts:
#	_data/projects.yml
